client/src/tests/questions.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow, mount, render } from 'enzyme';
+import Questions from '../Components/Questions.jsx';
+
+// These tests are not passing, I may have to pass in mock data for the questions props?
```

Comment:
hmm, is it giving you a specific error?

Reply:
I think it has to do with the fact that my component maps over some props. If I don't pass in any real data I don't think it runs...

<Questions /> › it should call a function when the next page button is clicked

    TypeError: this.props.questions.map is not a function

      218 |     return (
      219 |       <Body className='questionModule'>
    > 220 |         {this.props.questions.map((question, questionIndex) => {
          |                               ^
      221 |           if (questionIndex >= this.state.questionMin && questionIndex < this.state.questionMax) return (
      222 |             <QuestionModule key={questionIndex} className='question'>
      223 |               <Question>